hack/test-dnat-snat-baremetal.sh

```
@@ -0,0 +1,288 @@
+# ---------------------------------------------------------------------------
+cleanup() {
+  local node
+  say "  removing iptables rules from masters..."
+  for node in $(master_nodes); do
+    oc_host "$node" bash -c '
+      while iptables -t nat -L PREROUTING -n --line-numbers 2>/dev/null | grep -q "DNAT.*dpt:80"; do
+        line=$(iptables -t nat -L PREROUTING -n --line-numbers | awk "/DNAT.*dpt:80/ {print \$1; exit}")
+        [[ -n "$line" ]] || break
+        iptables -t nat -D PREROUTING "$line" || break
+      done
+      iptables -t nat -D POSTROUTING -p tcp -d '"${INGRESS_VIP}"'/32 --dport 80 -j MASQUERADE 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)